etebase_server/urls.py

```diff
@@ -1,15 +1,14 @@
 import os
 
 from django.conf import settings
-from django.conf.urls import url
 from django.contrib import admin
 from django.urls import path, re_path
 from django.views.generic import TemplateView
 from django.views.static import serve
 from django.contrib.staticfiles import finders
 
 urlpatterns = [
-    url(r"^admin/", admin.site.urls),
+    re_path(r"^admin/", admin.site.urls),
```

Comment:
```suggestion
    path("admin/", admin.site.urls),
```
Double check me, but I think this is preferred.

Reply:
Yep, looks like it, I just blindly followed the deprecation message ;)

I also converted the `re_path` below to `path`, apparently that one can capture URL parts as well: https://docs.djangoproject.com/en/3.0/ref/urls/#path

Reply:
I just confirmed the `admin/` route works with `path` as well as `re_path`.